docs/blog/_posts/2020-08-31-27th-dotty-milestone-release.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.27.0-RC1 - ScalaJS, performance, stability
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-08-31
+---
+
+Hello! We are excited to announce 0.27.0-RC1 of Dotty. In this version, we bring ScalaJS support to Dotty. As we are getting closer to the Scala 3 release, we continue shifting our focus to stability and performance, which are the central theme of this release.
+
+You can try out this version right now, from the comfort of your SBT, by visiting the [home page](https://dotty.epfl.ch/) and scrolling down to the "Create a Dotty Project" section.
+
+Alternatively, you can try this version of Scala online via [Scastie](https://scastie.scala-lang.org/). Once you're there, click "Build Settings" and set "Target" to "Dotty".
+
+Enjoy the ride🚀!
+
+<!--more-->
+# Welcome ScalaJS support
+PR [#9637](https://github.com/lampepfl/dotty/pull/9637) brings ScalaJS support to Dotty. Now, if you use the ScalaJSPlugin in your Dotty SBT project, the project will be correctly compiled to compile to JavaScript.
+
+# Stability
+As we're getting closer to the release of Scala 3, we are continuing to focus on the stability and performance of the language. In this release, we have fixed support of objects under JDK9 (PR [#9181](https://github.com/lampepfl/dotty/pull/9181)). The issue was, due to the changes in JDK9 compared to JDK8, our initialization scheme for objects did not work under JDK9. The aforementioned fixed that issue, thereby unblocking JDK9 support for Dotty.
+
+We are also continuing to work on stabilising enums. PR [#9532](https://github.com/lampepfl/dotty/pull/9532) corrects the deserialization and serialization of singleton enum values with `ObjectInputStream` and `ObjectOutputStream`. PR [#9549](https://github.com/lampepfl/dotty/pull/9549) enables overriding the `toString` method on enums – previously this was not possible because of the way enums were desugared.
```

Comment:
/cc @bishabosha

Reply:
In addition to https://github.com/lampepfl/dotty/pull/9532 I merged https://github.com/lampepfl/dotty/pull/9612 which is actually correct when mixing class and value cases

Reply:
Is that other PR related to serialization-deserialization? Or how should we mention it?

Reply:
Thats right, #9612 an implementation of deserialisation that completely replaces the implementation of #9532 so we should probably mention that one only. #9532 also has a bug that I didn't test (ordinal does not correspond to position in the values array) but is fixed in #9612

Reply:
although #9532 can be mentioned for enabling the values array even when enums have a type parameter